thanos/charts/alerts/compactor.yaml

```diff
@@ -0,0 +1,77 @@
+groups:
+- name: thanos-compact
+  rules:
+  - alert: ThanosCompactMultipleRunning
+    annotations:
+      description: No more than one Thanos Compact instance should be running at once.
+        There are {{$value}} instances running.
+      runbook_url: https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md#alert-name-thanoscompactmultiplerunning
+      summary: Thanos Compact has multiple instances running.
+    expr: sum by (job) (up{job=~".*thanos-compact.*"}) > 1
+    for: 5m
+    labels:
+      severity: warning
```

Comment:
We've introduced a consistent structure for creating alerts. Users will likely modify some fields via option values, which we can make accessible. Could you please use the [kubernetes-operations](https://github.com/cloudoperators/kubernetes-operations/blob/main/charts/kubernetes-operations/alerts/kubernetes-node.yaml) repository as a blueprint?

This could also be a follow-up task. It's important that these alerts can take additional labels like `support_group` and `service` to route them to the appropriate team.

Reply:
Note: The `runbook_url` can be moved to an alert label `playbook` to attach it to alert notifications. 
>> https://github.com/cloudoperators/greenhouse-extensions/pull/809/commits/07ef496b75b4fc06055c229b0057e5d9acc6d4f8

Reply:
adapted the structure of the labels, and added possibility to pass labels as option values